src/pal/src/include/pal/process.h

```diff
@@ -43,6 +44,11 @@ extern DWORD gSID;
 
 extern LPWSTR pAppDir;
 
+// The Mac sandbox application group ID (if exists) and container (shared) path
+extern LPCSTR gApplicationGroupId;
+extern int gApplicationGroupIdLength;
+extern PathCharString gApplicationContainerPath;
```

Comment:
It is confusing to call it `gApplicationContainerPath` when the name matches the reality only in sandbox on OSX. For the remaining cases (OSX without sandbox other Unixes), it is the global temp folder. Maybe `g_sharedMemoryFilesPath` would be more general.

Reply:
Im not convinced this will only be used by shared memory.  It will be used also by the debugger for creating the pipes.  Generally anything that needs to be shared by processes.  

Perhaps SharedFilesPath ?

Reply:
SharedFilesPath sounds good to me.